internal/services/compute/windows_virtual_machine_scale_set_extensions_test.go

```diff
@@ -407,6 +407,7 @@ resource "azurerm_windows_virtual_machine_scale_set" "test" {
     type                       = "CustomScriptExtension"
     type_handler_version       = "1.10"
     auto_upgrade_minor_version = true
+    automatic_upgrade_enabled  = true
```

Comment:
rather than adding this to all of the existing tests - since this behaviour is optional we should add a specific test for this field, so can we revert this and add a specific test instead?